packages/teams-js/src/public/pages.ts

```diff
@@ -752,4 +752,91 @@ export namespace pages {
       return runtime.supports.pages ? (runtime.supports.pages.appButton ? true : false) : false;
     }
   }
+
+  /**
+   * Provides functions for navigating without needing to specify your application ID.
+   *
+   * @beta
+   */
+  export namespace currentApp {
+    /**
+     * Parameters for the NavigateWithinApp
+     *
+     * @beta
+     */
+    export interface NavigateWithinAppParams {
+      /**
+       * Use {@link app.PageInfo.id | app.Context.page.id}
```

Comment:
I don't think app.Context.page.id actually goes anywhere (same with app.Context.page.subPageId). Does that link actually resolve to a place in code?

Reply:
`app.Context.page.id` and `app.Context.page.subPageId` is the link text. Reference link - https://docs.w3cub.com/jsdoc/tags-inline-link.html